src/actions.js

```diff
@@ -12,3 +12,9 @@ export const reset = formName => ({type: RESET, formName})
 
 export const DELETE_FIELD = 'c2-form/DELETE_FIELD'
 export const deleteField = (formName, field) => ({type: DELETE_FIELD, formName, field})
+
+export const DELETE_CURRENT_FORM = 'c2-form/DELETE_CURRENT_FORM'
+export const deleteCurrentForm = formName => ({type: DELETE_CURRENT_FORM, formName})
+
+export const DELETE_TARGET_FORM = 'c2-form/DELETE_TARGET_FORM'
+export const deleteTargetForm = targetForm => ({type: DELETE_TARGET_FORM, targetForm})
```

Comment:
How is DELETE_TARGET_FROM different from DELETE_CURRENT_FORM?  The both receive a formName as an argument.  I think we could combine the two into a single generic DELETE_FORM.  This would simplify the actions and reducers into one each since they all do the same thing.